addons/im_livechat/static/tests/embed/livechat_button_tests.js

```diff
@@ -22,10 +23,19 @@ test("open/close temporary channel", async () => {
 test("open/close persisted channel", async () => {
     await startServer();
     await loadDefaultConfig();
-    start();
+    await start({
+        mockRPC(route, args, originalRPC) {
```

Comment:
Isn't there onRpc something with HOOT now?

Reply:
in 17.2 live chat embed tests are still using qunit